src/garlic-toast.md

```diff
@@ -0,0 +1,27 @@
+# Garlic Toast
+
+Garlic toast, perfect as a substitute for crackers in a cheese platter.
+
+- ⏲️ Prep time: 5 min
+- 🍳Cook time: 10-15 min
+
+## Ingredients
+
+- Sourdough loaf (sliced)
+- Garlic-infused olive oil

```

Comment:
```suggestion
- Butter or Olive Oil (Optionally garlic-infused oil)
```
Butter also tastes good!

Reply:
you're probably not wrong, but it would hardly be garlic toast without the garlic-infused olive oil!